templates/boilerplate/package.json

```diff
@@ -30,6 +30,7 @@
     "msw": "^2.2.14",
     "react": "18.2.0",
     "react-native": "0.74.5",
+    "react-native-keyboard-aware-scroll-view": "^0.9.5",
```

Comment:
Since we are adding `react-native-keyboard-aware-scroll-view` do we still need the existing `react-native-keyboard-aware-scrollview`? (`scroll-view` vs `scrollview`)

Reply:
Good catch! [scroll-view](https://github.com/APSL/react-native-keyboard-aware-scroll-view) is actually what we want. It must have been a mistake to originally add [scrollview](https://github.com/wix-incubator/react-native-keyboard-aware-scrollview).